examples/complete/main.tf

```diff
@@ -0,0 +1,61 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 5.0.0"
+
+  name                 = local.vpc_name
+  cidr                 = var.vpc_cidr
+  private_subnets      = var.private_subnets
+  azs                  = var.availability_zones
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = {
+    Name        = "example-vpc"
+    Environment = "dev"
+  }
+}
+
+module "ecs_example" {
```

Comment:
please provide a meaningful name like ecs_app that describes what module you are importing